lib/rspec/core/pending.rb

```diff
@@ -51,36 +51,12 @@ class PendingExampleFixedError < StandardError; end
       #         raise "broken"
       #       end
       #     end
-      #
-      # @note `before(:example)` hooks are eval'd when you use the `pending`
-      #   method within an example. If you want to declare an example `pending`
-      #   and bypass the `before` hooks as well, you can pass `:pending => true`
-      #   to the `it` method:
-      #
-      #       it "does something", :pending => true do
-      #         # ...
-      #       end
-      #
-      #   or pass `:pending => "something else getting finished"` to add a
-      #   message to the summary report:
-      #
-      #       it "does something", :pending => "something else getting finished" do
-      #         # ...
-      #       end
       def pending(message=nil)
         current_example = RSpec.current_example
 
         if block_given?
           raise ArgumentError, <<-EOS.gsub(/^\s+\|/, '')
-            |The semantics of `RSpec::Core::Pending#pending` have changed in
-            |RSpec 3. In RSpec 2.x, it caused the example to be skipped. In
-            |RSpec 3, the rest of the example is still run but is expected to
-            |fail, and will be marked as a failure (rather than as pending) if
-            |the example passes.
-            |
-            |Passing a block within an example is now deprecated. Marking the
-            |example as pending provides the same behavior in RSpec 3 which was
-            |provided only by the block in RSpec 2.x.
+            |Passing a block within an example is not supported.
```

Comment:
I'm not sure what changing this message gains us, the previous text still applies 🤔

Reply:
It felt overly verbose given that RSpec 2's behaviour is long gone, and passing a block to `pending` inside the example was causing it to raise, all such usages were long fixed.